src/components/Filters/Filters.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react'
+import TodoContext from '../../contexts/TodoContext'
+import Option from 'components/Option/Option'
+import Select from 'components/Select/Select'
+
+const Filters = () => {
+  const { setStatusTodo } = useContext(TodoContext)
+  const handleStatus = (event) => setStatusTodo(event.target.value)
```

Comment:
Para seguir la convención y hacerlo un poco más explícito, hay que agregarle el evento al nombre:

```suggestion
  const handleStatusFilterChange = (event) => setStatusTodo(event.target.value)
```